Description: Add platform detection support for ARM (64-bit)
 This patch adds platform detection support for the ARM (64-bit)
 architecture, also called arm64 or aarch64.
 .

Index: netw-ib-ox-ag-5.39.0/src/netwib-src/src/genemake
===================================================================
--- netw-ib-ox-ag-5.39.0.orig/src/netwib-src/src/genemake
+++ netw-ib-ox-ag-5.39.0/src/netwib-src/src/genemake
@@ -112,7 +112,7 @@ case $unamem in
   "ppc64le" | "powerpc64le" )
     NETWIBDEF_SYSARCH="ppc64le"
     ;;
-  arm* )
+  arm* | aarch64 )
     NETWIBDEF_SYSARCH="arm"
     ;;
   mips* )
